builtin/staker/validation/validation.go

```diff
@@ -132,26 +133,38 @@ func (v *Validation) renew(aggregations *delta.Renewal, hasDelegations bool) *de
 	queuedDecrease := big.NewInt(0).Set(v.QueuedVET)
 	v.WithdrawableVET = big.NewInt(0).Add(v.WithdrawableVET, v.PendingUnlockVET)
 	v.QueuedVET = big.NewInt(0)
-	pendingUnlock := v.PendingUnlockVET
 	v.PendingUnlockVET = big.NewInt(0)
 
 	v.CompleteIterations++
 
+	changeWeight := big.NewInt(0).Add(newLockedVET, aggregations.NewLockedWeight)
 	v.LockedVET = big.NewInt(0).Add(v.LockedVET, newLockedVET)
-	newLockedWeight := newLockedVET
-	if hasDelegations {
-		newLockedWeight = big.NewInt(0).Sub(newLockedWeight, pendingUnlock)
-	}
-
-	changeWeight := big.NewInt(0).Add(newLockedWeight, aggregations.NewLockedWeight)
 	v.Weight = big.NewInt(0).Add(v.Weight, changeWeight)
 
 	// deltas
+	weight := stakes.NewWeightedStake(newLockedVET, Multiplier).Weight()
+	if delegationWeight.Sign() < 1 {
+		// no delegations - if there is a validator weight multiplied by 2 will be set to 1
+		weight = big.NewInt(0).Sub(weight, big.NewInt(0).Sub(v.Weight, v.LockedVET))
+		v.Weight = v.LockedVET
```

Comment:
Think this is generally safer in case one of them is edited later, then both get edited since its a pointer 

```suggestion
		v.Weight = big.NewInt(0).Set(v.LockedVET)
```

Reply:
Fixed